WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -51,12 +51,20 @@ open class WordPressOrgXMLRPCApi: NSObject {
         let sessionManager = Alamofire.SessionManager(configuration: sessionConfiguration)
 
         let sessionDidReceiveChallengeWithCompletion: ((URLSession, URLAuthenticationChallenge, @escaping(URLSession.AuthChallengeDisposition, URLCredential?) -> Void) -> Void) = { [weak self] session, authenticationChallenge, completionHandler in
-            return self?.urlSession(session, didReceive: authenticationChallenge, completionHandler: completionHandler)
+            guard let strongSelf = self else {
```

Comment:
@jkmassel I think it's sufficient to not return the result here. The return type is `Void` anyway.